docs/README.md

```diff
@@ -1,58 +1,82 @@
-# Scylla documentation
+# ScyllaDB Documentation
 
-This is the repository for [Scylla documentation site](http://docs.scylladb.com/)
+This repository contains the source files for ScyllaDB Open Source documentation.
 
-To report an issue with the documentation, please use GitHub issues.
+- The `dev` folder contains developer-oriented documentation related to the ScyllaDB code base. It is not published and is only available via GitHub.
+- All other folders and files contain user-oriented documentation related to ScyllaDB Open Source and are sources for [docs.scylladb.com](https://docs.scylladb.com/).
 
-This repository accepts GitHub pull requests.
+To report a documentation bug or suggest an improvement, open an issue in [GitHub issues](https://github.com/scylladb/scylla/issues) for this project.
 
-**Send pull requests to the master branch, not gh-pages.  gh-pages will be overwritten by deploy without warning.**
+To contribute to the documentation, open a GitHub pull request.
 
+## Key Guidelines for Contributors
 
-## Prerequisites
+- You need to fork this repository.
+- Work on your fork's task branches, not the `master` branch.
+- Send pull requests to the `master` branch in this repository.
+- If you want to submit a knowledge base article, see [Submitting a KB Article](#submitting-a-kb-article).
+- Follow the [ScyllaDB Style Guide](https://docs.google.com/document/d/1lyHp1MKdyj0Hh3NprNFvEczA4dFSZIFoukGUvFJb9yE/edit?usp=sharing).
+
+The user documentation is written in reStructuredText (RST) - a plaintext markup language similar to Markdown. If you're not familiar with RST, see [ScyllaDB RST Examples](https://sphinx-theme.scylladb.com/stable/examples/index.html).
+
+The developer documentation is written in Markdown. See [Basic Markdown Syntax](https://www.markdownguide.org/basic-syntax/) for reference.
+
+
+## Submitting a KB Article
+
+If you are submitting a knowledgebase article (KB), use the following guidelines:
+* In the `/kb_common` directory, there is a template for KBs. It is called `kb-article-template.rst`.
+* Make a copy of the KB template in the `/kb` directory and rename it with a unique name.
+* Open the new file and fill in the required information. 
+* Remove what is not needed. 
+* Run `make preview` to build the docs and preview them locally.
+* Send a PR with "KB" in its title. 
+
+
+## Building User Documentation
+
+### Prerequisites
 
 * Python 3. Check your version with `$ python --version`
-* Vale CLI (optional to lint docs). [Install Vale](https://docs.errata.ai/vale/install) for your operative system.
+* Vale CLI (optional to lint docs). [Install Vale](https://docs.errata.ai/vale/install) for your OS.
```

Comment:
@tzach Do we need it or can I remove it?

Reply:
Not sure. @dgarcia360 please advise

Reply:
Removed (confirmed by David https://github.com/scylladb/scylla/pull/11134#discussion_r931334730)